src/logius/splitmarkdownheaders.js

```diff
@@ -0,0 +1,123 @@
+// @ts-check
+/*
+ * Logius addition
+ * this script splits markdown level 1 headers ("#") in seperate sections
```

Comment:
Note that if you hit bugs with the markdown parser itself, you can also file bugs at:
https://github.com/markedjs/marked